tests/integrationv2/conftest.py

```diff
@@ -10,6 +18,15 @@ def pytest_configure(config):
         "markers", "uncollect_if(*, func): function to unselect tests from parametrization"
     )
 
+    no_pq = config.getoption('no-pq', 0)
+    fips_mode = config.getoption('fips-mode', 0)
+    if no_pq is 1:
+        set_flag(S2N_NO_PQ, True)
+    if fips_mode is 1:
+        set_flag(S2N_FIPS_MODE, True)
+
+    set_flag(S2N_OPENSSL_VERSION, config.getoption('openssl-version', 'openssl-1.1.1'))
```

Comment:
Why is the default 'openssl-1.1.1'? Can we not have a default, and verify that the version is always set if we're using the openssl provider?